lib/core/Head.js

```diff
@@ -95,6 +95,44 @@ class Head extends React.Component {
             title={this.props.config.title + ' Blog RSS Feed'}
           />
         )}
+        {this.props.config.gaTrackingId &&
+          this.props.config.gaGtag && (
+            <script
+              async
+              src={`https://www.googletagmanager.com/gtag/js?id=${
+                this.props.config.gaTrackingId
+              }`}
+            />
+          )}
+        {this.props.config.gaTrackingId &&
+          this.props.config.gaGtag && (
+            <script
```

Comment:
Couldn't use combine the the two `<script>` into one `gaTrackingID && gaGtag` check, instead of having two of the same check?

Reply:
Basically


```
        {this.props.config.gaTrackingId &&
          this.props.config.gaGtag && (
            <script
              async
              src={`https://www.googletagmanager.com/gtag/js?id=${
                this.props.config.gaTrackingId
              }`}
            />
            <script
              dangerouslySetInnerHTML={{
                __html: `
              window.dataLayer = window.dataLayer || [];
              function gtag(){dataLayer.push(arguments); }
              gtag('js', new Date());
              gtag('config', '${this.props.config.gaTrackingId}');
            `,
              }}
            />
          )}
```

Reply:
Does that not work?